config/initializers/okcomputer.rb

```diff
@@ -40,10 +40,41 @@ def profile
   end
 end
 
+# check models to see if at least they have some data
+class TablesHaveDataCheck < OkComputer::Check
+  def check
+    msg = [
+      Bookmark,
+      BulkAction,
+      Search,
```

Comment:
In the case of searches, this is okay to not have any. In fact they are supposed to be cleared out periodically: https://github.com/projectblacklight/blacklight/blob/91d5e6fb1fe8589262c3b86c0e5a2b5963e1f431/lib/railties/blacklight.rake#L5-L7